vulkaninfo/vulkaninfo.cpp

```diff
@@ -738,7 +757,10 @@ int main(int argc, char **argv) {
 
     // Combinations of output: html only, html AND json, json only, human readable only
     for (int i = 1; i < argc; ++i) {
-        if (strncmp("--json", argv[i], 6) == 0 || strcmp(argv[i], "-j") == 0) {
+        if (strcmp(argv[i], "--vkconfig_output") == 0) {
+            human_readable_output = false;
```

Comment:
suggest a comment here so it's clear this is intentionally private: e.g.:
"A internal-use-only format for communication with the Vulkan Configurator tool"